docs/docsite/rst/common_return_values.rst

```diff
@@ -84,6 +84,10 @@ warnings
 ````````
 This key contains a list of strings that will be presented to the user.
 
+deprecations
+````````````
+This key contains a list of dictionaries that will be presented to the user. Keys of the dictionaries are `msg` and `version`, values are string, value for the `version` key can be an empty string.
```

Comment:
I am not sure if the documentation should describe the format used after of before call to [`_return_formatted`](https://github.com/pilou-/ansible/blob/fcdcd5931953ce42e9009264a7fa2372af28bb7b/lib/ansible/module_utils/basic.py#L1938) ?